css/src/components/verticalNav.css

```diff
@@ -4,7 +4,7 @@
 }
 
 .VerticalNav--expanded {
-  width: var(--spacing-9);
+  width: 240px;
```

Comment:
Why we are setting width here again when we have already given width to its parent?

Reply:
Because the parent is Collapsible and that is according to the scrollbar but we need it to be 240px even without the scrollbar